inductiva/resources/machines.py

```diff
@@ -0,0 +1,107 @@
+"""Containes Machines class to manage resources."""
+
+from absl import logging
+from typing import Optional
+from uuid import UUID
+import uuid
+import inductiva
+from inductiva import api
+
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.instance_api import InstanceApi
+from inductiva.client.model.instance import Instance
+from inductiva.client.model.instance_group import InstanceGroup
+
+
+class Machines():
+    """Class to Google Cloud manage resources."""
```

Comment:
typo? word order... ?
Maybe you meant "Class to manage Google Cloud resources."?